src/harness/certs/generate_fake_certs.sh

```diff
@@ -150,6 +180,60 @@ openssl ca -cert unknown_ca.cert -keyfile private/unknown_ca.key -in unknown_dev
     -policy policy_anything -extensions cbsd_req_sign -config ../../../cert/openssl.cnf \
     -batch -notext -create_serial -utf8 -days 1185 -md sha384
 
+#Certificate for test case WINNF.FT.S.SCS.12 - Expired certificate presented during registration
+echo "\n\nGenerate 'client_expired' certificate/key"
+openssl req -new -newkey rsa:2048 -nodes \
+    -reqexts cbsd_req -config ../../../cert/openssl.cnf \
+    -out client_expired.csr -keyout client_expired.key \
+    -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=SAS CBSD Example"
+openssl ca -cert cbsd_ca.cert -keyfile private/cbsd_ca.key -in client_expired.csr \
+    -out client_expired.cert -outdir ./root \
+    -policy policy_anything -extensions cbsd_req_sign -config ../../../cert/openssl.cnf \
+    -batch -notext -create_serial -utf8 -startdate 20150214120000Z -enddate 20160214120000Z -md sha384
+
+#Certificate for test case WINNF.FT.S.SCS.15 - inapplicable fields certificate presented during registration
+echo "\n\nGenerate 'inapplicable certificate for WINNF.FT.S.SCS.15' certificate/key"
+openssl ca -cert cbsd_ca.cert -keyfile private/cbsd_ca.key -in client.csr \
+    -out client_inapplicable.cert -outdir ./root \
+    -policy policy_anything -extensions cbsd_req_inapplicable_sign -config ../../../cert/openssl.cnf \
+    -batch -notext -create_serial -utf8 -days 1185 -md sha384
+
+#Certificate for test case WINNF.FT.S.SCS.16 -
+openssl ca -revoke cbsd_ca.cert -keyfile private/root_ca.key -cert root_ca.cert \
+     -config ../../../cert/openssl.cnf
+
+echo "\n\n Generate CRL for root_ca"
+openssl ca -gencrl -keyfile private/root_ca.key -cert root_ca.cert \
+     -config ../../../cert/openssl.cnf  -crlhours 1\
+     -out root/crl/root_ca.crl
+
+echo "\n\n Generate CRL for cbsd_ca"
+openssl ca -gencrl -keyfile private/cbsd_ca.key -cert cbsd_ca.cert \
+     -config ../../../cert/openssl.cnf -crlhours 1 \
+     -out root/crl/cbsd_ca.crl
+
+# Certificate for WINNF.FT.S.SCS.17,WINNF.FT.S.SCS.18 and WINNF.FT.S.SCS.19
```

Comment:
It might be better to create this cert in the test case itself so that it can run automatically. Right now, the tester would need to create the cert and then quickly run the specified test.

Reply:
The test   cases related to this comment has been removed. We will make it a   configurable test case when it gets submitted.